helm/kpack-image-builder/templates/cluster-builder.yaml

```diff
@@ -3,6 +3,8 @@ apiVersion: kpack.io/v1alpha2
 kind: ClusterStore
 metadata:
   name: cf-default-buildpacks
+  annotations:
+    korifi.cloudfoundry.org/helm-release-version: {{ .Release.Revision | quote }}
```

Comment:
Is that annotation related?

Reply:
Ah no. That was me trying to fix yesterday's kpack problems. I'll remove it.